### PR TITLE
ascanrules: Add more example alerts

### PR DESCRIPTION
**Pull request by kingthorin**
## Overview
- CHANGELOG > Add notes.
- Scan rules > Add example alert functionality (6119).
- Unit tests > Assert the new example alerts.

## Related Issues
- zaproxy/zaproxy#6119

## Checklist
- [NA] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [x] Write tests
- [x] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

### CODE REVIEW

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteCodeExecutionCve20121823ScanRule.java [hunk 1]
```diff
@@ -168,20 +169,7 @@ private boolean scan(URI originalURI, URI attackURI, String payload) {
...
                 LOGGER.debug("Remote Code Execution alert for: {}", originalURI);
 
-                // bingo.
-                newAlert()
-                        .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                        .setDescription(
-                                Constant.messages.getString(
-                                        "ascanrules.remotecodeexecution.cve-2012-1823.desc"))
-                        .setAttack(payload)
-                        .setOtherInfo(responseBody)
```
Comment (thc202) [inline addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteCodeExecutionCve20121823ScanRule.java:178]: Better mention in the changelog.

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ServerSideIncludeScanRule.java [hunk 2]
```diff
@@ -203,4 +206,9 @@ public int getCweId() {
     public int getWascId() {
         return 31;
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(buildAlert("profile", SSI_UNIX, patternSSIUnix.toString()).build());
```
Comment (thc202) [inline addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ServerSideIncludeScanRule.java:212]: The evidence is not the pattern but what was matched by the pattern.

File: addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/RemoteCodeExecutionCve20121823ScanRuleUnitTest.java [hunk 3]
```diff
@@ -179,9 +180,10 @@ void shouldAlertIfWindowsAttackWasSuccessful() throws Exception {
                                 "<?php exec('cmd.exe /C echo "
                                         + RemoteCodeExecutionCve20121823ScanRule.RANDOM_STRING
                                         + "',$colm);echo join(\"\n\",$colm);die();?>")));
+        System.out.println(alertsRaised.get(0).getAttack());
+        System.out.println("**" + body);
```
Comment (thc202) [inline addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/RemoteCodeExecutionCve20121823ScanRuleUnitTest.java:184]: Remove.

File: addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ServerSideIncludeScanRule.java (hunk 2 above)
Comment (kingthorin) [reply addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ServerSideIncludeScanRule.java:212]: Oops the String Builder toString call mislead me 🤦‍♂️

**Comment by kingthorin** [comment]
Done x3

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**